packages/manager/modules/pci/src/projects/project/gateways/add/add.html

```diff
@@ -97,6 +97,26 @@
             data-display-selected-region="$ctrl.displaySelectedRegion"
             data-on-change="$ctrl.onRegionChange(region)"
         ></pci-project-regions-list>
+        <div
+            class="warning-text pb-2"
+            data-ng-if="$ctrl.selectedRegion && !$ctrl.selectedRegion.enabled && $ctrl.selectedRegion.inactive"
+        >
+            <span
+                data-translate="pci_projects_project_public_gateways_add_region_activate_line1"
+            ></span>
+            <a
+                data-ng-href="{{ $ctrl.regionsLink }}"
+                target="_self"
+                rel="noopener"
+            >
+                <span
+                    data-translate="pci_projects_project_public_gateways_add_region_activate_line2"
+                ></span>
+            </a>
+            <span
+                data-translate="pci_projects_project_public_gateways_add_region_activate_line3"
+            ></span>
+        </div>
```

Comment:
**niptick:** Use `oui-message` instead of `div` with classes.

Reply:
Done